openadapt/config.py

```diff
@@ -38,8 +56,73 @@ def getenv_fallback(var_name):
 ROOT_DIRPATH = pathlib.Path(__file__).parent.parent.resolve()
 DB_FPATH = ROOT_DIRPATH / DB_FNAME
 DB_URL = f"sqlite:///{DB_FPATH}"
+DIRNAME_PERFORMANCE_PLOTS = "performance"
 
 if multiprocessing.current_process().name == "MainProcess":
     for key, val in locals().items():
         if not key.startswith("_") and key.isupper():
             logger.info(f"{key}={val}")
+
+
+# ACTION EVENT CONFIGURATIONS
+TEXT_SEP = "-"
+TEXT_NAME_PREFIX = "<"
+TEXT_NAME_SUFFIX = ">"
```

Comment:
Please define these in `_DEFAULTS`.

Also please prefix with `ACTION_`

Reply:
Done!

Reply:
But this will cause pylint error: `Access to a protected member _DEFAULTS of a client class Pylint(W0212:protected-access)`

when calling them in another module, eg. config._DEFAULTS.get("ACTION_TEXT_SEP")

Reply:
Just use `config.ACTION_TEXT_SEP`

Reply:
Alright!